presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlCaseInsensitiveMapping.java

```diff
@@ -41,20 +40,18 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        this.postgreSqlServer = new TestingPostgreSqlServer();
+        postgreSqlServer = new TestingPostgreSqlServer();
+        closeAfterClass(() -> {
+            postgreSqlServer.close();
+            postgreSqlServer = null;
```

Comment:
when good order is now maintained,`... = null` is excessive, it doesn't help much (or even at all) now, but hurts readability a lot, `postgreSqlServer = closeAfterClass(new ...)` is enough

Reply:
It hurts readability, but it is sanity check that. So it is impossible to access `postgreSqlServer` in not usable state. Alternatively we could mantain state in `TestingPostgreSqlServer` to know that is `CREATED` or `STARTED` or `CLOSED` but this is even worse.